catalog/imaging/catalog-imaging-transformer/src/main/java/com/connexta/alliance/transformer/nitf/NitfPreStoragePlugin.java

```diff
@@ -95,12 +95,21 @@ private void process(List<ContentItem> contentItems) {
         ContentItem contentItem = contentItems.get(0);
         Metacard metacard = contentItem.getMetacard();
 
+        if (contentItem.getMimeTypeRawData() == null || !contentItem.getMimeTypeRawData()
+                .equals(NitfInputTransformer.MIME_TYPE)) {
+            LOGGER.debug("skipping content item: filename={} mimeType={}",
+                    contentItem.getFilename(),
+                    contentItem.getMimeTypeRawData());
+            return;
+        }
+
```

Comment:
Understood.  We'll create a story to do this after the Alliance 2.9.0 (or 2.9.1) release, then?